modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -2,17 +2,17 @@
 :description: Manage the throughput of Kafka traffic with configurable properties.
 :page-categories: Management, Networking
 
-Manage the throughput of Kafka traffic at the cluster level, with configurable properties that limit and protect the use of disk and network resources for individual brokers and for an entire cluster. Set broker-wide throughput limits for Kafka API traffic.
+Redpanda allows cluster administrators to configure throughput limits at both the broker level and the client level. For brokers, these configurable limits help protect the use of disk and network resources. Limits applied on the broker apply to all clients that produce or consume there. Alternatively, the client limits allow you to selectively throttle a single client or group of clients. These limits help protect individual clients from 'noisy neighbors' that may consume extreme amounts of available throughput and crowd out other clients.
 
 == Broker-wide throughput limits
 
 The network bandwidth and disk utilization of brokers may be overloaded by clients that produce or consume throughput without limits. To prevent resource overloading caused by unconstrained throughput and to configure back pressure, Redpanda provides runtime-configurable properties that limit and balance throughput of Kafka API traffic.
 
 To manage the volume of traffic going through a broker, Redpanda implements throughput quotas on the ingress and egress sides of every broker. The throughput quota accounts for all Kafka API traffic going in or out of a broker, with the value of quota representing the allowed rate of data passing through in one direction. When a connection is in breach of the quota, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level, and it implements that delay before handling Kafka API requests. To control the quotas, Redpanda provides configurable rate limits for total ingress and egress traffic through a broker.
 
-With Redpanda's xref:get-started:architecture.adoc#thread-per-core-model[thread-per-core model], the Kafka API traffic to and from a client connection is processed by a single core (shard). In order to manage throughput quotas efficiently, broker quotas are distributed between shards, and each per-shard quota is in turn shared by all connections served by the shard. Splitting broker quota optimally between shards is done behind the scenes by the quota balancer component.
+== Advanced throughput limit tuning
```

Comment:
Much better, but I think the advanced section should go below the properties tables.  the properties are the basic stuff, this stuff should be at the bottom.  And I assume we should say 'under normal circumstances these should not need tuning'.  True @BenPope ?

Reply:
I think reordering this is perfectly fair, especially since we're not referencing a lot of these advanced properties in that table. I was trying to make sure it didn't get confused with the topic-level throughput features but I think it'll be clear.